cmd/kn/main.go

```diff
@@ -62,6 +64,11 @@ func run(args []string) error {
 
 	pluginManager := plugin.NewManager(config.GlobalConfig.PluginsDir(), config.GlobalConfig.LookupPluginsInPath())
 
+	// Merge plugins that implements client-pkg Plugin type to the same manager instance
+	for _, p := range pkgplugin.InternalPlugins {
+		pluginManager.AppendPlugin(p)
+	}
+
```

Comment:
This part is important! 

There're 2 gloabal variables: 
- `client` `InternalPlugins`
- `client-pkg` `InternalPlugins`

Depending in the plugin impl both are still valid, either by using new depedency or former on client repo. Therefore there's list merge to a single one that is used onward. I.e. content of `client-pkg` list is merged into `client` list.